deploy/crds/kataconfiguration.openshift.io_v1alpha1_kataconfig_cr.yaml

```diff
@@ -2,6 +2,9 @@ apiVersion: kataconfiguration.openshift.io/v1alpha1
 kind: KataConfig
 metadata:
   name: example-kataconfig 
+spec:
+  config:
+     sourceImage: quay.io/isolatedcontainers/kata-operator-payload:4.5.4
```

Comment:
This was supposed to be vanilla k8s support feature only. We do not want users to specify the source image in OpenShift. Instead in OpenShift it should infer the correct image to use from the cluster version.

Reply:
It's right, we should just infer the right version from the OCP version. But: I thought about this, I still think it is a useful feature even for the OpenShift version for testing. If a user chooses this in production I'd just consider it not supported. Which is something we should document. What do you think?